### PR TITLE
Updating version for the next release

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

Updates AGW version to `1.9`

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by saltiyazan — APPROVED** [review]

File: python/magma_access_gateway_installer/agw_installer.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 
 class AGWInstaller:
-    MAGMA_VERSION = "focal-1.8.0"
+    MAGMA_VERSION = "focal-1.9.0"
```
Comment (gruyaume) [inline python/magma_access_gateway_installer/agw_installer.py:14]: 1.9? Does it even exist?

Comment (Gmerold) [reply python/magma_access_gateway_installer/agw_installer.py]: Ups... 
Reverted.

**Review by ghislainbourgeois — APPROVED** [review]

File: python/setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 setup(
     name="magma-access-gateway-installer",
-    version="1.8.0",
+    version="1.9.0",
```
Comment (gruyaume) [inline python/setup.py:19]: Still there :eyes:

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: magma-access-gateway
 base: core20
-version: '1.8.0'
+version: '1.9.0'
```
Comment (gruyaume) [inline snap/snapcraft.yaml:3]: Another one

File: python/setup.py (hunk 2 above)
Comment (Gmerold) [reply python/setup.py:19]: `main` is now meant to hold the next release. We can't be building `1.8` from both `main` and the `1.8` branch. 
For now they'll be the same, but eventually, we'll bring in 1.9 changes here.

File: snap/snapcraft.yaml (hunk 3 above)
Comment (Gmerold) [reply snap/snapcraft.yaml:3]: Please see the comment above.

File: python/setup.py (hunk 2 above)
Comment (Gmerold) [reply python/setup.py:19]: This version here doesn't point to any upstream release. It defines our component's version which is related to the snap version. Upstream package version that we actually install is defined in the `agw_installer.py` and it currently points to `focal-1.8.0`.

**Pull request merged**